examples/mongodb/jest.config.js

```diff
@@ -0,0 +1,13 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
```

Comment:
you might need `transformIgnorePattern`, see https://github.com/facebook/jest/blob/5442d34c9ba2123b122c5f3e9230943ed9074d1a/e2e/global-setup/package.json#L4-L7

Reply:
Yep, that appears to be the issue.